newgistics.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "faraday", "~> 1.10"
   spec.add_dependency "tzinfo", "~> 2.0"
 
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", "~> 2.1"
```

Comment:
This may be questionable, but I couldnt dev locally without it.

Reply:
We use bundler 2.1.x on the main `quip` anyway, so this makes sense